src/django/api/serializers.py

```diff
@@ -103,16 +104,33 @@ def get_contributor_id(self, user):
 
     def get_claimed_facility_ids(self, user):
         if not switch_is_active('claim_a_facility'):
-            return []
+            return {
+                'approved': None,
+                'pending': None,
+            }
 
         try:
-            return FacilityClaim \
+            approved = FacilityClaim \
                 .objects \
                 .filter(status=FacilityClaim.APPROVED) \
                 .filter(contributor=user.contributor) \
                 .values_list('facility__id', flat=True)
+
+            pending = FacilityClaim \
+                .objects \
+                .filter(status=FacilityClaim.PENDING) \
+                .filter(contributor=user.contributor) \
+                .values_list('facility__id', flat=True),
+
+            return {
+                'pending': list(chain(*pending)),
```

Comment:
This is here to flatten the lists; somehow they became nested like [[123], [456]] when making this change.

Reply:
We were bitten by the Python! There is a trailing comma just above at the end of the `pending =` assignment which ended up creating a tuple. I was able to remove that comma and the list(chain(*)) wrappers and see things working as expected.

Reply:
Wow, thanks for catching that! I dropped the trailing comma and the `list(chain` call and everything worked as before. Will amend the commit to include the changes before merging.